pkgs/data/icons/windows10-icons/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchurl }:
+stdenv.mkDerivation rec {
+  pname = "windows10-icons";
+  version = "1.0";
+
+  src = fetchurl {
+    url = "https://github.com/B00merang-Artwork/Windows-10/archive/refs/tags/${version}.tar.gz";
+    sha256 = "sha256-ZGuLyOBNOANIicqs7zhGy2ZVmV8YBdh7P/e8Cykkmq4=";
+  };
+
+  installPhase = ''
+    install -Dm644 $(find . -type f -maxdepth 1) -t $out/share/icons/windows10
+    for dir in $(find . -type d -maxdepth 1)
+      do install -d $dir $out/share/icons/windows10/$dir
+    done
+  '';
+
+  meta = with lib; {
+    description = "Windows 10 icon theme";
+    homepage = "http://b00merang.weebly.com/windows-10.html";
+    license = licenses.unfree;
+    maintainers = with maintainers; [ mib ];
```

Comment:
Please add your user to https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix first.

See https://nixos.org/manual/nixpkgs/unstable/#var-meta-maintainers

Reply:
My first PR #237651  - where I do add myself to the maintainers list - is going through the review process right now. Is it best to wait till it gets merged and FF this branch or should I add it to this PR too? (my concern is mainly merge conflicts, but if it's no problemo then I'll just copy the commit :))

Reply:
Whoops, sorry. Totally missed the comment where you already mentioned that^^

Since you asked:
You could base this PR on top of https://github.com/NixOS/nixpkgs/pull/237651/commits/189e0ded75b46a9ab4bed91cb213744eae92e6ad from the other PR.

But it shouldn't matter all that much. The easiest would be to just wait for the other PR to get merged first :)

Reply:
#237651 has been merged. So I believe this change has been solved. :smile:

Reply:
Totally right! I'll resolve and get the PR ready ^^